includes/wc-cart-functions.php

```diff
@@ -369,7 +369,19 @@ function wc_cart_round_discount( $value, $precision ) {
 	if ( version_compare( PHP_VERSION, '5.3.0', '>=' ) ) {
 		return round( $value, $precision, WC_DISCOUNT_ROUNDING_MODE );
 	} else {
-		return round( $value, $precision );
+		// Fake it in PHP 5.2.
+		if ( 2 === WC_DISCOUNT_ROUNDING_MODE ) {
+			$value    = (string) $value;
+			$value    = explode( '.', $value );
+			$value[1] = substr( $value[1], 0, $precision + 1 );
```

Comment:
I think this needs a check to make sure $value[1] exists. There are a bunch of undefined offset errors in 5.2 now from this line. https://travis-ci.org/woocommerce/woocommerce/jobs/262615644